fv/host_port_test.go

```diff
@@ -146,7 +148,21 @@ var _ = infrastructure.DatastoreDescribe("with initialized Felix", []apiconfig.D
 			})
 
 			It("port should be reachable", func() {
-				Eventually(metricsPortReachable, "10s", "1s").Should(BeTrue())
+				//Eventually(metricsPortReachable, "10s", "1s").Should(BeTrue())
+				// Switching to the loop below, *seems* to have fixed a flake
+				// that was happening here with the Eventually check above.
+				// The loop is purposefully high vs what is being checked so
+				// if another failure occurs we can see if the timeout is just
+				// a little too low or it is something more.
+				start := time.Now()
+				for time.Since(start).Seconds() < 50 {
+					reachable := metricsPortReachable()
```

Comment:
Suggest you wrap this with ```
startTime=time.Now()
...
log.WithField("time", time.Since(startTime)).Info("Checked metrics port")
```
so that we find out exactly how long it took  (unless you're sure we can already work that out from the logs).